cloudinary/api.py

```diff
@@ -154,6 +154,19 @@ def delete_derived_resources(derived_resource_ids, **options):
     return call_api("delete", uri, params, **options)
 
 
+def delete_derived_by_transformation(derived_resource_ids, transformation, **options):
+    uri = ["derived_resources"]
+    if not isinstance(derived_resource_ids, list):
+        derived_resource_ids = [derived_resource_ids]
+    params = [("derived_resource_ids[]", derived_resource_id) for derived_resource_id in derived_resource_ids]
+    if not isinstance(transformation, list):
+        transformation = [transformation]
+    for t in transformation:
```

Comment:
Instead of an array, serialize transformations with `build_eager` see [ruby example](https://github.com/cloudinary/cloudinary_gem/blob/2f95b77c561f2dda91f2bd69fbc2b7d7cf32ebd9/lib/cloudinary/api.rb#L157).